auto_dev/data/repo/templates/autonomy/scripts/pre_commit_hook.sh

```diff
@@ -12,9 +12,9 @@ function check_output_command() {
 }
 
 function process() {
-    adev -n 0 -v fmt -co 
+    poetry run adev -n 0 -v fmt -co 
     check_output_command "Error in formatting the code"
-    adev -n 0 -v lint -co
+    poetry run adev -n 0 -v lint -co
```

Comment:
Unnecessary as should already be in a venue to run the hooks

Reply:
contradicts your suggestion here https://github.com/8ball030/auto_dev/pull/346#discussion_r1842743174
&& poetry run is more explicit and reliable.   pydantic uses similar for their pre-commit